lib/dotcom/trip_plan/input_form.ex

```diff
@@ -49,6 +49,7 @@ defmodule Dotcom.TripPlan.InputForm do
       date: PlanParams.to_date_param(datetime),
       time: PlanParams.to_time_param(datetime),
       transportModes: __MODULE__.Modes.selected_mode_keys(modes) |> PlanParams.to_modes_param(),
+      numItineraries: 40,
```

Comment:
I kinda wanna let this slide, but 40 seems kinda excessive?

Reply:
Oh oops - didn't mean to include this change. I've been using that to get more itineraries so that _the one I'm looking for_ is more likely to actually show up.